scylla-gdb.py

```diff
@@ -3789,6 +3789,25 @@ def invoke(self, arg, for_tty):
             f = reference_wrapper(f).get()
             gdb.write('%s: %s\n' % (f['_name'], f['_enabled']))
 
+class scylla_repairs(gdb.Command):
```

Comment:
Please document the command, see e.g. `scylla fiber` for an example.

Reply:
ok.